### DISCUSSION TITLE
Machine readable files--permitted file types question

### DISCUSSION THREAD

**Discussion by jnc108park** [CMSgov/hospital-price-transparency#66] [Q&A] [answered]
Can we use the XLS file type for the MRF for the time being, until we need to switch over to the CMS template layout as of July 1, 2024? We are in the process of switching over and having technical issues moving away from our current use of the XLS file type.

**Comment by RidenourK** [comment, marked as answer]
@jnc108park Thank you for your question. 

Machine-readable file means a single digital file that is in a machine-readable format. 

As finalized in the CY2024 OPPS/ASC final rule, beginning July 1, 2024, all hospitals must adopt a CMS template and encode its standard charge information in it in the form and manner finalized in the [Hospital Price Transparency Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency). CMS has made the CMS template available in three non-proprietary formats: CSV Tall, CSV Wide, and JSON.

**Reply by dmercer-sch** [reply]
@RidenourK I don't think you understood @jnc108park's question as your answer does not answer their question.

**Reply by jnc108park** [reply]
I would like to know if the only formats (until July 1, 2024) are CSV Tall, CSV Wide, and JSON, or are there others, so long as it is a single digital file in a machine-readable format.  This is my exact question. Thank you!

**Reply by RidenourK** [reply]
@jnc108park Thank you for your question. 

This GitHub discussion section is intended for technical questions or suggestions about the CMS templates or schema that are required as of July 1, 2024, suggestions for additions to the valid values for the templates, and solicitation input from other users of the template about their approach. Please direct other questions about Hospital Price Transparency policy and requirements to CMS at [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov). If your hospital is under compliance review, direct your inquiries to [HPTCompliance@cms.hhs.gov](mailto:HPTCompliance@cms.hhs.gov).
